saleor/plugins/base_plugin.py

```diff
@@ -1017,39 +1017,55 @@ def _update_config_items(
         config_structure: dict = (
             cls.CONFIG_STRUCTURE if cls.CONFIG_STRUCTURE is not None else {}
         )
+        configuration_to_update_dict = {
+            c_field["name"]: c_field["value"] for c_field in configuration_to_update
```

Comment:
The previous code used `.get()`. Are we sure, those keys will be in dict?

Reply:
Thanks for catching that!